packages/emf/backend/sirius-components-interpreter/src/main/java/org/eclipse/sirius/components/interpreter/AQLInterpreter.java

```diff
@@ -112,12 +114,38 @@ public AQLInterpreter(List<Class<?>> classes, List<Object> instances, List<EPack
         ePackages.stream().filter(this::isValidEPackage).forEach(ePackage -> {
             String nsURI = ePackage.getNsURI();
             long start = System.nanoTime();
-            this.queryEnvironment.registerEPackage(ePackage);
+            AtomicBoolean cacheHit = new AtomicBoolean();
```

Comment:
You don't really need an atomic boolean, you can just use a regular boolean easily

Reply:
I think it was needed in an earlier version of the code that was structured differently, but yes, not anymore.